.github/workflows/test.yml

```diff
@@ -29,32 +49,38 @@ jobs:
         uses: actions/cache@v3
         with:
           path: "$HOME/.foundry/cache"
-          key: rpc-cache-${{ hashFiles('cli/tests/it/integration.rs') }}
+          key: rpc-cache-${{ hashFiles('cli/tests/rpc-cache-keyfile') }}
 
       - name: cargo test
-        run: cargo test --locked --workspace --all-features --lib --bins
+        run: cargo test --locked --workspace --all-features --lib --bins fork
 
-  doc:
-    name: doc tests
+  integration:
+    name: integration tests
     runs-on: ubuntu-latest
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
+
       - name: Install toolchain
         uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
           profile: minimal
           override: true
-      - uses: Swatinem/rust-cache@v1
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v1
         with:
           cache-on-failure: true
-
+      # required for forge commands that use git
+      - name: setup git config
+        run: |
```

Comment:
using `git config --global` can solve the failed tests related to `forge init`
i tested it [here](https://github.com/0xYYY/foundry/runs/6594279127?check_suite_focus=true)